src/batch/benches/hash_agg.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
+use futures::StreamExt;
+use risingwave_batch::executor::test_utils::{gen_data, MockExecutor};
+use risingwave_batch::executor::{BoxedExecutor, HashAggExecutorBuilder};
+use risingwave_batch::task::TaskId;
+use risingwave_common::catalog::schema_test_utils::field_n;
+use risingwave_common::types::DataType;
+use risingwave_expr::expr::AggKind;
+use risingwave_pb::batch_plan::HashAggNode;
+use risingwave_pb::expr::agg_call::Arg;
+use risingwave_pb::expr::{AggCall, InputRefExpr};
+use tikv_jemallocator::Jemalloc;
+use tokio::runtime::Runtime;
+
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
+
+fn mock_task_id() -> TaskId {
+    TaskId {
+        task_id: 0,
+        stage_id: 0,
+        query_id: String::new(),
+    }
+}
+
+fn create_hash_agg_executor(chunk_size: usize, chunk_num: usize) -> BoxedExecutor {
```

Comment:
Please add more data types for grouping and agg functions, and make it easy to extend.

Reply:
Does it look better now?